packages/compiler/src/templates/Panels.svelte

```diff
@@ -0,0 +1,27 @@
+ <script>
+ </script>
+
+<style>
+  .container {
+	  margin-left: auto;
+	  margin-right: auto;
+	  padding-left: 15px;
+	  padding-right: 15px;
+	  width: 100%;
+  }
+  .row {
```

Comment:
I suspect we want to get this class from bootstrap.

Reply:
Removed styling from Panels in e815cfd995850ae42c63a72287c5c4b9db517ca8, should be defaulted to Bootstrap styles once #205 is merged.